test/server/parse-fixture-test.js

```diff
@@ -38,8 +49,8 @@ function testFixtureParser (dir, expectedStructure, fileParser, codeParser) {
 
         return fileParser(filePath)
             .then(function (structure) {
-                expect(structure).eql(expected);
-                expect(codeParser(fileContent)).eql(expected);
+                lazyEql(structure, expected);
```

Comment:
The `lazyEql` is not necessary now. You can use the `expect().eql()` expression again.